src/Navigation.re

```diff
@@ -34,7 +34,7 @@ external addListener:
   EventSubscription.t =
   "addListener";
 
-// TODO: setParams, ...
+[@bs.send] external setParams: (t, 'a) => unit = "";
```

Comment:
I would use `Js.t('a)` instead of `'a` here

Reply:
Yes, It makes sense. And for the sake of https://github.com/reason-react-native/reason-react-navigation/pull/11 I will add name too.